Imager.js

```diff
@@ -142,30 +143,36 @@
           });
         }
 
-        if (elements) {
-            this.divs = applyEach(elements, returnFn);
-            this.selector = null;
-        }
-        else {
-            this.divs = applyEach(doc.querySelectorAll(this.selector), returnFn);
-        }
-
+        this.divs = [];
+        this.add(elements || this.selector);
         this.ready(opts.onReady);
-        this.changeDivsToEmptyImages(this.divs);
 
-        setTimeout(function(){
+        setTimeout(function () {
             self.init();
         }, 0);
-    }
+    };
+
+    Imager.prototype.add = function (elementsOrSelector) {
+
+        elementsOrSelector = elementsOrSelector || this.selector;
```

Comment:
Ah yes indeed, I somewhat had in mind the existing classes were forwared to the responsive image… which is not the case.

We can survive with the oddity for the moment then :-)

Thanks!